src/test/java/io/shapelets/khiva/LibraryTest.java

```diff
@@ -67,7 +86,84 @@ public void testGetDevice() {
     }
 
     @Test
-    public void testGetKhivaVersion() {
-        Assert.assertEquals(Library.getKhivaVersion(), "0.1.0");
+    public void testGetKhivaVersion() { Assert.assertEquals(Library.getKhivaVersion(), getKhivaVersionFromGithub()); }
```

Comment:
Using this way of testing the version is fine when we download the latest from the khiva C++ repo, compile it and then test the Java implementation. However, if anyone has a previous version of Khiva installed on his/her computer and then run the tests, it might lead to a test failure if there is a newer tag in the khiva C++ repo. Just saying